File: CTDK_Python/113/_113.py
n = int(input("Nhap n: "))
if (n == 1):
    ahh = 2
else:
    at = 2
    i = 2

    while (i <= n):
        ahh = at + 2 * i + 1
        i += 1
        at = ahh

print("Ket qua:", ahh)

File: CTDK/113/113.cpp
#include <iostream>
using namespace std;

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	int ahh = 0;
	int at = 2;
	int i = 2;
	while (i <= n)
	{
		ahh = at + 2 * i + 1;
		i++;
		at = ahh;
	}
	cout << "So hang thu " << n << " : " << ahh << endl;
	return 0;
}